domains/svg.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "svg's website",
+    "repo": "https://github.com/svg-rs/svg.github.io",
+    "owner": {
+      "username": "svg-rs",
+      "email": "kimmkayy8@proton.me"
+    },
+    "record": {
+      "CNAME": "svg.github.io"

```

Comment:
Please change the record value to `svg-rs.github.io`.

Reply:
ok

Reply:
i fixed it
can you merge noww